src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -132,24 +147,29 @@ if [ ! -z ${RunCrossGen2+x} ]%3B then
         $__R2RDumpCommand
         __r2rDumpExitCode=$?
 
-        export DOTNET_GCName DOTNET_GCStress DOTNET_HeapVerify DOTNET_ReadyToRun DOTNET_TC_OnStackReplacement DOTNET_TC_PartialCompilation
+        # FIXME: Run PdbChecker if enabled
+
+        export DOTNET_GCName DOTNET_GCStress DOTNET_HeapVerify DOTNET_ReadyToRun
+        export DOTNET_TC_OnStackReplacement DOTNET_TC_PartialCompilation
         date +%H:%M:%S
       }
 
       if [ ! -z ${CompositeBuildMode+x} ]%3B then
-          ExtraCrossGen2Args+=" --composite"
-          OneFileCrossgen2 "$PWD/composite-r2r.dll" "$PWD/IL-CG2/*.dll"
+          shopt -s nullglob
+          __CrossgenInputsList=($PWD/IL-CG2/*.dll)
```

Comment:
What is the benefit of passing the inputs list in a global variable instead of the original way that was passing the argument?

Reply:
It spits them all into a single line in the response file that way, and then crossgen fails to parse the multiple file names in one line. This is the only thing I came up with that worked. It's possible the variable isn't needed and I can pass an array, I'm not good with bash.

Reply:
You can pass it in the old way and then in the function do this:
```sh
    arg2=$2
    printf "%s\n" ${arg2[@]} >> $__ResponseFile
```
or
```sh
   arg2 = $2
   for filename in ${arg2[@]}
   do
        echo $filename >> $__ResponseFile
   done
```

Reply:
Unfortunately passing it in the old way doesn't pass the entire contents of the glob as an array in $2. It passes the files sequentially, i.e. $2 $3 $4 $5

Reply:
That is quite strange, I've tested it with a simple test script and it worked properly. Have you put quotes around the argument? That's important for this to work.

Reply:
I mean, at the call site of the function.

Reply:
I didn't put quotes around the argument at the call site, that would do the trick I think, now that you mention it. Will try it.